backend/app/api.py

```
@@ -2095,6 +2095,242 @@ def delete_slot_threshold(current_user, room_db_id, slot_id, threshold_id):
+        ).all()
+        
+        groups = []
+        for (member_key,) in members:
+            try:
+                # Try structured JSON list decoding
+                parsed = json.loads(member_key)
+                if isinstance(parsed, list) and len(parsed) == 2:
+                    g_name, item = parsed
+                    if item.lower() == item_name.lower():
+                        groups.append(g_name)
+            except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)